robotics_integration_tests/utilities/flotilla_backend_api.py

```diff
@@ -501,3 +540,45 @@ def wait_for_robot_status(
             )
             time.sleep(1)
             continue
+
+
+def pause_mission(backend_url: str, robotId: str) -> None:
```

Comment:
```suggestion
def pause_mission(backend_url: str, robot_id: str) -> None:
```

Naming convention should be snake_case